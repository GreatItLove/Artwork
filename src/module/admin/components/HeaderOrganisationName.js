import React from 'react';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';

const propTypes = {
  organization: PropTypes.string.isRequired
};

/**
 * @function HeaderOrganisationName
 * @param {String} organization
 * @returns {XML}
 */
function HeaderOrganisationName({ organization }) {
  return <h6>{organization}</h6>;
}

const mapStateToProps = state => ({
  organization: state.customer.organization
});

HeaderOrganisationName.propTypes = propTypes;

export { HeaderOrganisationName as HeaderOrganisationNameTest };

export default connect(mapStateToProps)(HeaderOrganisationName);
