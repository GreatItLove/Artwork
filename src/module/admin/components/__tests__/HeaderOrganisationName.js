import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import { HeaderOrganisationNameTest as HeaderOrganisationName } from '../HeaderOrganisationName';

describe('HeaderOrganisationName', () => {
  it('Should be same snapshot', () => {
    const wrapper = shallow(<HeaderOrganisationName organization="Test organisation name" />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
