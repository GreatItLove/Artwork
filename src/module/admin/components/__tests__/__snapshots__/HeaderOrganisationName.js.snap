// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderOrganisationName Should be same snapshot 1`] = `
<h6>
  Test organisation name
</h6>
`;
